test/specs/basic.js

```diff
@@ -0,0 +1,9 @@
+const assert = require('assert');
+
+describe('Main page asana', () => {
+  it('should have the right title', () => {
+    browser.url('https://asana.com/es');
+    const title = browser.getTitle();
+    assert.strictEqual(title, 'Gestiona en línea el trabajo, los proyectos y las tareas de tu equipo · Asana');
+  });

```

Comment:
we were supposed to create at least two test cases one for a success log in and the other when cannot log in the app